jest.config.js

```diff
@@ -2,7 +2,11 @@ module.exports = {
   collectCoverage: true,
   // Ensures that we collect coverage from all source files, not just tested
   // ones.
-  collectCoverageFrom: ['./src/**/*.ts', '!./src/**/*index.ts'],
+  collectCoverageFrom: [
+    './src/**/*.ts',
+    '!./src/**/*index.ts',
+    '!./src/js/colors/colors.ts',
```

Comment:
Ignoring coverage for the deprecated colors file as it just uses the objects from light and dark theme color which there are already tests for